include/dslash_quda.h

```diff
@@ -77,22 +77,27 @@ namespace quda
      * @return shmem_sync_t
      */
     shmem_sync_t get_shmem_sync_counter();
-
+    shmem_sync_t get_shmem_sync_counter2();
```

Comment:
why do we need a v2 (and for the set below)?

Reply:
the 2 version is used exclusively for the ghost exchange. Probably worth to come up with a better name here. sync_counter is used in the fine dslash. Maybe dslash_sync_counter and ghost_sync_counter ?

Reply:
Yes, that sounds like a good name change suggestion